Islandora-CLAW/islandora_demo
# use edtf field type

**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/998

# What does this Pull Request do?

Updates date fields for repository item to use the new EDTF field type.

# What's new?

* Changes date_edtf, date_edtf_created, date_edtf_issued to use the edtf date type

# How should this be tested?

- Clean claw-playbook will throw an error:
```
TASK [Install demo module (should not fail)] ***********************************
Wednesday 30 January 2019  16:06:48 -0800 (0:00:03.454)       0:42:15.327 ***** 
fatal: [default]: FAILED! => {"changed": true, "cmd": ["/var/www/html/drupal/vendor/drush/drush/drush", "-y", "en", "islandora_demo"], "delta": "0:00:05.056578", "end": "2019-01-31 00:06:51.620452", "msg": "non-zero return code", "rc": 1, "start": "2019-01-31 00:06:46.563874", "stderr": "\nIn DiscoveryTrait.php line 52:\n                                          \n  The \"text_edtf\" plugin does not exist.  \n                                          \n\npm:enable [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-d|--debug] [-y|--yes] [--no] [--remote-host REMOTE-HOST] [--remote-user REMOTE-USER] [-r|--root ROOT] [-l|--uri URI] [--simulate] [--pipe] [-D|--define DEFINE] [--druplicon] [--notify [NOTIFY]] [--xh-link XH-LINK] [--] <command> [<modules>]...", "stderr_lines": ["", "In DiscoveryTrait.php line 52:", "                                          ", "  The \"text_edtf\" plugin does not exist.  ", "                                          ", "", "pm:enable [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-d|--debug] [-y|--yes] [--no] [--remote-host REMOTE-HOST] [--remote-user REMOTE-USER] [-r|--root ROOT] [-l|--uri URI] [--simulate] [--pipe] [-D|--define DEFINE] [--druplicon] [--notify [NOTIFY]] [--xh-link XH-LINK] [--] <command> [<modules>]..."], "stdout": "", "stdout_lines": []}
```
- wipe out the box `vagrant destroy -f`
- update the inventory ('inventory/group_vars/webserver/drupal.yml') to use the PR branch: `islandora/islandora_demo:isue-998`
- `vagrant up --provision` should finish clean


# Interested parties
@Islandora-CLAW/committers
